test/unittests/iotools/utils/test_data_handling.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from string import Template
+from typing import Union
+
+import pytest
+
+
+def test_vox_to_world_space():
+    """Test function `vox_to_world_space_method_1`."""
```

Comment:
Maybe add a comment to explicit what we are trying to test (i.e that the return data structure is of the right format)